Assets/VRTK/SDK/Daydream/DaydreamReach.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// DaydreamReach component uses touchpad to extend controller position out in front of user
+/// along the current controller orientation plane
+/// </summary>
+public class DaydreamReach : MonoBehaviour
+{
+    // controller to track, defaults to ./Controller but probably want a Pointer Joint like GvrControllerPointer/Laser
+    public Transform controller;
+
+    // maximum reach distance from controller origin
+    public float reachDistance = 0.5f;
+
+    private Vector3 positionOrigin;
+
+    void Start()
```

Comment:
monobehaviour methods should all be protected virtual to allow script extension if required.